File: bluetooth.py
#!/usr/local/bin/python3
import serial
from firebase import firebase
import time

APP='https://arduinobt.firebaseio.com'
PORT='/dev/tty.HC-06-DevB'
RATE=9600

f = firebase.FirebaseApplication(APP,None)
s = serial.Serial(port=PORT, baudrate=RATE)

while True:
	try:
		web_value = str(f.get('/light-value',None))
		print (web_value)
		s.write(str(web_value).encode())
		time.sleep(0.5)

	except (KeyboardInterrupt, EOFError):
		print()
		print("Closing serial...")
		s.close()
		break
		print()

File: bluetooth.js
var f = new Firebase('https://arduinobt.firebaseio.com/');

function toggle() {
	f.on('value', function(x) {
		var value = x.val()
		console.log(value);
	});
};
/*
		f.set({
			"light-value": value===
		});

		document.getElementById('current').innerHTML = value;
*/